### PR TITLE
fix: distinguish header and logo colors, added proper logo styles

### LINKED ISSUE #484: [Design]: Colour distinction between h1 and logo
### Describe the bug.

as mentioned, currently the header and the logo have the same colour code which make the distinction between them hard additionally the logos are mostly of text.

### Expected behavior

darken out the header colour to make the distinction visible

### Screenshots

image

### How to Reproduce

1. homepage footer

### 🥦 Browser

Brave

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Tanmayshi**
fix: distinguish header and logo colors, added proper logo styles
Improvement: Updated the logo colors to align with the proper color scheme for better visual clarity.
Additional Change: Enhanced the UI by improving the contrast between header and logo elements, making it more user-friendly.
Related issue(s)
Fixes #484

### CODE REVIEW

File: components/Sponsors/sponsors.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function Sponsors({eventSponsors,financialSponsor}) {	 //eventSponsors,financial
 						orchestrate an unforgettable celebration.
 					</Paragraph>
 				</div>
-				<Heading typeStyle='heading-md' className='text-white mt-10 sm:text-2xl sm:mt-5'>Event and Host Sponsor</Heading>
+				<Heading typeStyle='heading-md' className='text-gray-800 mt-10 sm:text-2xl sm:mt-5'>Event and Host Sponsor</Heading>
```
Comment (TenzDelek) [inline components/Sponsors/sponsors.js:20]: a lighter shade is appreciated. (also try with stale)

File: pages/index.js [hunk 2]
```diff
@@ -186,7 +186,7 @@ export default function Home() {
 			</div>
 			</div>
 			<div id='sponsors' className='mt-20'>
-				<Sponsors eventSponsors={[{image:'/img/apidays.png',websiteUrl:"https://www.apidays.global/"}]} financialSponsor={[{image:"/img/graviteeio.svg" , websiteUrl: "https://www.gravitee.io/",className:"w-[250px] h-[50px]" } , {image:"/img/postman.png" ,websiteUrl:"https://www.postman.com/",className:"w-[240px] h-[70px]"}]}  />
+				<Sponsors eventSponsors={[{image:'/img/apidays_2.webp',websiteUrl:"https://www.apidays.global/"}]} financialSponsor={[{image:"/img/graviteeIo_1.png" , websiteUrl: "https://www.gravitee.io/",className:"w-[250px] h-[50px]" } , {image:"/img/postman_1.png" ,websiteUrl:"https://www.postman.com/",className:"w-[240px] h-[70px]"}]}  />
```
Comment (TenzDelek) [inline pages/index.js:189]: I believe there is no need to change the logo, the logo are taken from the official branding i believe

**Review by AceTheCreator — CHANGES_REQUESTED** [review]
Kindly fix your PR title as suggested by the bot :)

**Comment by Tanmayshi** [comment]
"Hi @AceTheCreator,
I have updated the PR title as per the Conventional Commit guidelines (fix: distinguish header and logo colors, added proper logo styles). Please review the changes and let me know if there's anything else to address. 😊"

**Comment by AceTheCreator** [comment]
> "Hi @AceTheCreator, I have updated the PR title as per the Conventional Commit guidelines (fix: distinguish header and logo colors, added proper logo styles). Please review the changes and let me know if there's anything else to address. 😊"

I think you should apply suggested changes from @TenzDelek 🤔

**Comment by TenzDelek** [comment]
I think the contributor don't want to work on this issue looking at the deleted branch. Therefore I will be taking the issue from here

**Pull request closed without merging**